wikipedia-dark.user.css

```diff
@@ -735,9 +768,35 @@ regexp("https?:\/\/wiki\.(archlinux|mozilla)\.(org|jp)\/.*$") {
   tr > td[style="background: #CFE8FF;"], tr > td[style="background: #EED8AE;"],
   tr > td[style="background: #FFF8DC;"], tr > td[style="background: #B9D3FF;"],
   tr > td[style="background: #6495ED;"], tr > td[style="background: #607CD2;"],
-  tr > td[style="background: #828BD9;"], tr > td[style="background: #F0F8FF;"] {
+  tr > td[style="background: #828BD9;"], tr > td[style="background: #F0F8FF;"],
+  .infobox tr[style*="background:#F6E5A8;"] td,
+  .infobox tr[style*="background:#F6E5A8;"] th,
+  td[style*="background:#AED0F0"],
+  #mw-content-text .infobox-above[style*="background-color: lightblue"],
+  #mwAg .infobox-above[style*="background-color: lightblue"],
+  #mw-content-text .infobox-above[style*="background: #CCCCFF;"],
+  #mwAg .infobox-above[style*="background: #CCCCFF;"],
+  #mwAg .infobox-above[style*="background-color: lightblue"],
+  #mw-content-text .infobox-above[style*="background-color: #e7dcc3;"],
+  #mwAg .infobox-above[style*="background-color: #e7dcc3;"],
+  #mw-content-text .infobox-above[style*="background-color:  #c9ffd9;"],
+  #mw-content-text .infobox-above[style*="background:#F6DA9F"],
+  #mw-content-text .infobox tr[style*="background:#F8C9B0;"] td,
+  [class*="map"] [style*="font-size"][style*="left"],
+  .infobox [style*="position"][style*="width"][style*="top"][style*="font-size"],
+  .toccolours [style*="position"][style*="width"][style*="top"][style*="font-size"],
+  table[style*="background-color:transparent; "] span[style*="background-color:#FEFEE9;"],
+  .thumbimage [style*="position"][style*="width"] [style*="font-size"] {
     color: var(--gray-1) !important;
   }
+  tr[style*="background:#B9C5FF;"] > td,
+  td[style*="background: #FFCC66;"],
+  td [style="background: #FFFF99;"],
+  td[style="background: #4169E1;"],
+  td[style="background: #8AB0FF;"],
+  tr[style*="background:#E5D8C0;"] > td {
+    color: var(--gray-1);
```

Comment:
A good `npm run clean` and `npm run test` would ensure that these are formatted according to the style guide. I'll run them shortly anyway, but ideally the commit should have those corrections already.